components/layout/FavoriteActivityCard.js

```diff
@@ -0,0 +1,87 @@
+import Image from "next/image";
+import { FaHeart } from "react-icons/fa";
+import styled from "styled-components";
+
+export default function FavoriteActivityCard({
+  id,
+  title,
+  country,
+  categories,
+  imageUrl,
+  toggleBookmark,
+}) {
+  return (
+    <StyledFavoriteCard
+      style={{ backgroundImage: `url(${imageUrl})`, backgroundSize: "cover" }}
+    >
+      <CardHeader>
+        <TagsContainer>
+          {categories.map((category) => (
+            <StyledTag key={category}>{category}</StyledTag>
+          ))}
+        </TagsContainer>
+
+        <HeartContainer onClick={() => toggleBookmark(id)}>
+          <FaHeart style={{ color: "red" }} />
```

Comment:
Please note my comment regarding the icon. Please do not use inline styles or classes.